### ISSUE TITLE
Visible label and accessible name do not match in faq component

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/midd-frontend#711]
Siteimprove report - https://my2.siteimprove.com/Inspector/7333/A11Y/Page?pageId=72188742978&impmd=D97548790CE293BDC08F6D9B84D3E607&conf=a+aa+aria+si&issue=cantTell+failed&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r14/failed/d1l4R0pMak0=;ZjA0YmEzOGQ=
